runtime/src/accounts_index.rs

```diff
@@ -2491,8 +2491,8 @@ pub mod tests {
         assert_eq!(num, 1);
 
         // not zero lamports
-        let index = AccountsIndex::<AccountInfoTest, AccountInfoTest>::default_for_tests();
-        let account_info: AccountInfoTest = 0 as AccountInfoTest;
+        let index = AccountsIndex::<bool, bool>::default_for_tests();
```

Comment:
these were using cached entries, which have a refcount of 0. The new insertion mechanism only allows insertion of ref_count=1 entries. It is unnecessary to test (and illegal in real use) cached entries through `insert_new_if_missing_into_primary_index`